tests/test_analyzer.py

```diff
@@ -56,3 +58,11 @@ def test_analyzer_management(db, bad_db, cluster):
 
     # Test delete missing analyzer with ignore_missing set to True
     assert db.delete_analyzer(analyzer_name, ignore_missing=True) is False
+
+    # Test create geo_s2 analyzer (EE only)
+    if enterprise and db_version >= version.parse("3.10.5"):
+        analyzer_name = generate_analyzer_name()
+        result = db.create_analyzer(analyzer_name, "geo_s2", {})
+        assert result["type"] == "geo_s2"
+        assert result["properties"]["format"] == "latLngDouble"
```

Comment:
Could you please check other default values for this analyzer? Default options are the following:
```
{ 
  "options" : { 
    "maxCells" : 20, 
    "minLevel" : 4, 
    "maxLevel" : 23 
  }, 
  "type" : "shape", 
  "format" : "latLngDouble" 
}
```

Reply:
👍 added as of https://github.com/ArangoDB-Community/python-arango/pull/263/commits/6e2b00a461dc0072517c4e1c9d0a934da8763d85